### PR TITLE
Feature/app 5248 basic sync

### PR DESCRIPTION
**Pull request by john5macd**
Basic connection to INSSight to retrieve data

### CODE REVIEW

File: INSS.EIIR.DataSync.Infrastructure/Source/SQL/INSSightSQLSource.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            var dbContextOptsBldr = new DbContextOptionsBuilder<ExternalInssContext>();
+            dbContextOptsBldr.UseSqlServer(options.ConnectionString, sqlServerOptions => sqlServerOptions.CommandTimeout(60));
+
+            _externalInssContext = new ExternalInssContext(dbContextOptsBldr.Options);
+            this._options = options;
+        }
+
+        public async IAsyncEnumerable<InsolventIndividualRegisterModel> GetInsolventIndividualRegistrationsAsync()
+        {
+            await foreach (var x in _externalInssContext.VwEiirs.Where(p => p.InsolvencyType == "Bankruptcy" && p.HasRestrictions == 1).AsNoTracking().AsAsyncEnumerable())
```
Comment (VasInsolvencyService) [inline INSS.EIIR.DataSync.Infrastructure/Source/SQL/INSSightSQLSource.cs:30]: Use Constants for the string "Bankruptcy"

**Review by VasInsolvencyService — CHANGES_REQUESTED** [review]

File: INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    {
+        public static SyncData Get(IServiceProvider sp)
+        {
+            var factory = sp.GetRequiredService<ILoggerFactory>();
+            var config = sp.GetRequiredService<IConfiguration>();
+            var mapper = sp.GetRequiredService<IMapper>();
+
+            IEnumerable<IDataSourceAsync<InsolventIndividualRegisterModel>> sources = new List<IDataSourceAsync<InsolventIndividualRegisterModel>>()
+            {
+                GetINSSightSQLSource(config, mapper),
```
[the remark this answers is not in the document]
Comment (john5macd) [reply INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs:28]: Commented code will be required, added additional comments to explain why

File: INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                //GetEIIRSQLSource(config, mapper)
+            };
+
+            IEnumerable<IDataSink<InsolventIndividualRegisterModel>> sinks = new List<IDataSink<InsolventIndividualRegisterModel>>()
+            {
+                //GetXMLSink(config),
```
[the remark this answers is not in the document]
Comment (john5macd) [reply INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs:39]: Commented code will be required, added additional comments to explain why

File: INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+using INSS.EIIR.DataSync.Infrastructure.Sink.XML;
+using INSS.EIIR.DataSync.Infrastructure.Source.AzureTable;
+using INSS.EIIR.DataSync.Infrastructure.Source.SQL;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace INSS.EIIR.DataSync.Functions.DI
+{
+    public class SyncDataFactory
```
[the remark this answers is not in the document]
Comment (john5macd) [reply INSS.EIIR.DailyExtract/DI/SyncDataFactory.cs]: Changes class to static

File: INSS.EIIR.DataSync.Infrastructure/Source/SQL/INSSightSQLSource.cs (hunk 1 above)
Comment (john5macd) [reply INSS.EIIR.DataSync.Infrastructure/Source/SQL/INSSightSQLSource.cs:32]: Ultimately won't be required as this code is just temporary to reduce the dataset retrieved from INSSight in short term added a commentto explain why

**Review by VasInsolvencyService — APPROVED** [review]

**Pull request merged**